general-superstaq/general_superstaq/service.py

```diff
@@ -313,31 +332,49 @@ def submit_aces(
             num_circuits: How many random circuits to use in the protocol.
             mirror_depth: The half-depth of the mirror portion of the random circuits.
             extra_depth: The depth of the fully random portion of the random circuits.
-            kwargs: Other execution parameters.
-                - tag: Tag for all jobs submitted for this protocol.
-                - lifespan: How long to store the jobs submitted for in days (only works with right
+            method: Which type of method to execute the circuits with.
+            noise: Noise model to simulate the protocol with. Valid strings are
+                "symmetric_depolarize", "phase_flip", "bit_flip" and "asymmetric_depolarize".
+            error_prob: The error probabilities if a string was passed to `noise`.
+
+                For "asymmetric_depolarize", `error_prob` will be a three-tuple with the
+                error rates for the X, Y, Z gates in that order. So, a valid argument would be
+                `error_prob = (0.1, 0.1, 0.1)`. notice that these values must add up to less than
+                or equal to 1. For the other channels, `error_prob` is one number less than or
+                equal to 1, e.g., `error_prob = 0.1`.
```

Comment:
nits
```suggestion
                For "asymmetric_depolarize", `error_prob` will be a three-tuple with the error
                rates for the X, Y, Z gates in that order. So, a valid argument would be
                `error_prob = (0.1, 0.1, 0.1)`. Notice that these values must add up to less than
                or equal to 1. For the other channels, `error_prob` is one number less than or
                equal to 1, e.g., `error_prob = 0.1`.
```
also wdyt about formatting these as bullets? e.g.
```suggestion
                * For "asymmetric_depolarize", `error_prob` will be a three-tuple with the error
                rates for the X, Y, Z gates in that order. So, a valid argument would be
                `error_prob = (0.1, 0.1, 0.1)`. Notice that these values must add up to less than
                or equal to 1.
                * For the other channels, `error_prob` is one number less than or equal to 1, e.g.,
                `error_prob = 0.1`.
```

Reply:
Formatted as bullet points!